src/test/java/core/basesyntax/model/OperationTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import org.junit.Test;
+
+public class OperationTest {
+
+    @Test
+    public void getBalanceEnum_Ok() {
```

Comment:
```suggestion
    public void getByCode_balanceOperation_ok() {
```
pattern for naming of test methods `(method that you test)_(case that you test)_(ok/notOk)`. Fix naming of test methods in every test class